docs/USING.md

```diff
@@ -10,12 +10,47 @@ At the moment, we would not recommend using Mars for servers that rely on perfec
 
 ### Using
 
-Download the latest Mars build [here](https://nightly.link/Warzone/mars/workflows/build/master/Mars.zip).
+1. Download the latest Mars build. You can find this by going to the [actions tab](https://github.com/Warzone/mars/actions) and navigating to the latest successful build.
 
-On a PGM server ([setup guide](https://github.com/PGMDev/PGM/blob/dev/docs/RUNNING.md)), add the Mars plugin JAR file to the server's `plugins/` folder. After starting the server once, Mars will generate [a config file](https://github.com/Warzone/mars/blob/master/src/main/resources/config.yml) which you need to modify to match the API (API URLs and secret token). Once the config is updated, restart the server and Mars should be working (you should see a connection message in the API console).
+2. Download the latest [Mars API](https://github.com/Warzone/mars-api) build. You can find this by going to the [actions tab](https://github.com/Warzone/mars-api/actions) and navigating to the latest successful build.
```

Comment:
Now that new features in Mars are exclusively backed by the Rust implementation of the API, we need to reflect that here -- maybe mention both implementations as options but recommend the Rust one for availability of new features, and mention the last version of Mars that works with the Kotlin API?

Reply:
archiving the old api and only encouraging the use of the rust API feels like the best approach. there might be changes made in the future that might break support in the future and in case that happens, discouraging its use is probably the best idea. maybe not even mention the old one and only link to the rust api.